WTWSMS/history/characters/greek.txt

```diff
@@ -3108,7 +3108,7 @@
 }
 
 1310016 = {
-	name="Solomon" #Solomon 4 Duke of Sirmium, don�t forget to expand his land
+	name="Solomon" #Solomon 4 Duke of Sirmium
```

Comment:
There is a reason behind this, and it is still valid. The titles this character held are currently not represented.